authors/settings.py

```diff
@@ -162,8 +162,8 @@
     'DEFAULT_AUTHENTICATION_CLASSES': (
         'authors.apps.authentication.backends.JWTAuthentication',
     ),
-    'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.LimitOffsetPagination'
-
+    'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.LimitOffsetPagination',
+    'PAGE_SIZE': 9
```

Comment:
Lets wait and see if the PO has an opinion on this default limit. The rest is ok. Do you intend to add any unit tests for this?

Reply:
Now that I have added the default limit, I think I need to test it.